### PR TITLE
New lane for easier build incrementation after release.

### PR DESCRIPTION
**Pull request by jankrzempek**
The idea here was to improve the process of changing the version after release.
For some time we are dealing with a situation when changing the version in one place does not change it in all others. I was trying not to use external tools when doing this lane, but after hours of trying, I think it's reasonable to give it a chance as it is doing its job well.

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -151,6 +151,25 @@ platform :ios do
...
+         version_number: new_version
+      )
+      increment_build_number(
+         build_number: "0"
+      )
+      puts "Setting a version number to #{new_version} and reseting build to #{get_build_number}"
+      sh("git checkout -b bumped_version_to_#{new_version}")
+      sh("git add -A")
+      sh("git commit -m \"Change version number to #{new_version} and build reset\"")
+      sh("git push -u origin bumped_version_to_#{new_version}")
```
Comment (pawelgil) [inline fastlane/Fastfile:170]: I'm on the fence when it comes to pushing from fastlane. Not sure that's a good idea

**Review by pawelgil — APPROVED** [review]
great stuff!! 👏👏👏

**Comment by jankrzempek** [comment]
> Looks cool! 👏🏻 Did you test it after the last release?

Not exactly after the release, but I tested it few times, and seems to be working fine

File: fastlane/Fastfile (hunk 1 above)
Comment (jankrzempek) [reply fastlane/Fastfile:170]: I am going to delete the push and extend the repo description then. Could you tell me why could be not a good idea?

**Pull request merged**